### ISSUE TITLE
MacOS M1

### ISSUE THREAD

**Issue by ademaro** [janvarev/Irene-Voice-Assistant#3]
Так как vosk на данный момент не поддерживает m1, соответственно, в зависимостях он не ставится вообще. Можно запустить через Rosetta (и поставить x86_64 python).

Что бы запустить я поставил отдельно hombrew (для Rosetta приложений): https://stackoverflow.com/questions/64951024/how-can-i-run-two-isolated-installations-of-homebrew/64951025#64951025

Далее на маках по умолчанию голоса другие, поэтому совершенно непонятно что читается (видимо, как на линуксе), но это немного другая история, оформлю в отдельной ветке для маков чуть позже...

**Comment by trankov** [comment]
На сегодняшний день Vosk декларирует поддержку M1 у себя на сайте в разделе Installations.

**Comment by ademaro** [comment]
@trankov да, спасибо, действительно встало на 3.10 m1, на днях буду тестить)

Подробности тут:
https://github.com/alphacep/vosk-api/issues/692
